exercises/bracket-push/analysis_options.yaml

```diff
@@ -0,0 +1,18 @@
+analyzer:
+  strong-mode:
+    implicit-casts: false
+    implicit-dynamic: false
+  errors:
+    unused_element:        error
+    unused_import:         error
+    unused_local_variable: error
+    dead_code:             error
+
+linter:
+  rules:
+      # Error Rules
+      - avoid_relative_lib_imports
+      - avoid_types_as_parameter_names
+      - literal_only_boolean_expressions
+      - no_adjacent_strings_in_list
+      - valid_regexps
```

Comment:
Recommend you format this yaml file. There are two extra spaces on the lines above